recipes/shadow-dom-native/cypress/integration/file-input.spec.js

```diff
@@ -46,4 +46,19 @@ describe('Attach file to a shadow dom input element', () => {
         .shadowContains('cy-2.png');
     });
   });
+
+  it('successfully uploads an empty file when allowEmpty is true', () => {
+    cy.fixture('empty.txt', 'base64').then(fileContent => {
```

Comment:
it looks like it's not possible to upload a *not* empty file when `allowEmpty` is `true`?

Reply:
See comment https://github.com/abramenal/cypress-file-upload/pull/119#issuecomment-551436057 above, and subsequent back and forth.

Reply:
`allowEmpty` to me it means that an empty file can also be uploaded apart from the default behavior. Otherwise the prop should be renamed to `allowEmptyOnly` to avoid confusions.

Reply:
If we’re going to revisit it, I’d rather it was changed back to the original allowEmpty intent, but as I said above, I don’t feel strongly about it.